frontend/src/components/NavBar/index.js

```diff
@@ -3,10 +3,14 @@ import PropTypes from "prop-types";
 import { AppBar, Button, Toolbar } from "@mui/material";
 import HomeIcon from "@mui/icons-material/Home";
 import { CampaignName, LogoButton } from "./navBar.styled";
+import BasicMenu from "./NavBarMenu";
 
 const NavBar = (props) => {
   const { campaign } = props;
 
+  // FIXME: use actual logged in state
+  const isLoggedIn = true;
```

Comment:
this is fine for now, but when I would write code like this, I would simply write a util function in `src/utils/index.js` call it someething like `isLoggedIn` and then do 
```suggestion
const loggedIn = isLoggedIn()
```
this is more desirable, as the `isLoggedIn` function can just return true, but can be called from multiple different files. Ie, you only have to fix one function, rather than 10 differetn components that may have different `//FIXME` for logged in stuff.